src/test/java/io/github/eocqrs/kafka/consumer/settings/KfConsumerParamsTest.java

```diff
@@ -38,16 +38,12 @@ final class KfConsumerParamsTest {
 
   @Test
   void representsXmlCorrectly() {
+    final Params mock = Mockito.mock(Params.class);
+    Mockito.when(mock.asXml()).thenReturn("<groupId>103</groupId>");
```

Comment:
Case is simple one. But if we will have more complex scenario with dependencies we will need more mocking engine. Maybe instead of using the mocking frameworks such as Mockito we can build our own fake objects for that purposes